app/src/main/java/ch/epfl/sweng/swenggolf/offer/Offer.java

```diff
@@ -379,6 +395,7 @@ public Builder() {
             latitude = 0;
             linkPicture = "";
             uuid = UUID.randomUUID().toString();
+            isClosed = false;
```

Comment:
Can you add the the fact that the boolean is set to false in the javadoc ?

Reply:
It's done.